kornia/geometry/quaternion.py

```diff
@@ -244,6 +245,20 @@ def from_matrix(cls, matrix: Tensor) -> 'Quaternion':
         """
         return cls(rotation_matrix_to_quaternion(matrix, order=QuaternionCoeffOrder.WXYZ))
 
+    @classmethod
+    def from_angle_axis(cls, angle_axis: Tensor) -> 'Quaternion':
```

Comment:
```suggestion
    def from_axis_angle(cls, axis_angle: Tensor) -> 'Quaternion':
```
At some point we need to update the naming for this function too in conversions, let just start from here